src/LOR/models.py

```diff
@@ -1,6 +1,44 @@
 from django.db import models
+from django.utils.timezone import now
+
+
+#add proffesor in future, right now it is just a sample
+Proffesors= [
```

Comment:
this will have to change in the future as admins, writers, and requesters are all users